packages/core/database/lib/entity-manager/relations-orderer.js

```diff
@@ -87,17 +184,19 @@ const relationsOrderer = (initArr, idColumn, orderColumn) => {
       return this;
     },
     connect(relations) {
-      _.castArray(relations).forEach((relation) => {
+      const sortedRelations = sortConnectArray(relations, arr, strict);
+      sortedRelations.forEach((relation) => {
```

Comment:
Do we need to store this variable? 🤔 
Do you think it might need a check to know if the result of `sortConnectArray` function is a valid array?
```suggestion
      sortConnectArray(relations, arr, strict).forEach((relation) => {
```

Reply:
We could do:
```
sortConnectArray(_.castArray(relations), arr, strict).forEach((relation) => {
```

but I find it quite unreadable, do you know any other more readable form than this?

Reply:
I wouldn't mind checking if relations is an array with TS, but we don't do TS. 😅 
Maybe we can do a quick return check before calling sortConnectArray ?
Like `if (!Array.isArray(relations) || !relations.length) { return; }`